openmetadata-ui/src/main/resources/ui/debug.json

```diff
@@ -55,10 +55,7 @@
       "testFilePath": "/Users/shaileshparmar/Desktop/deuex/openmetadata-collate/OpenMetadata/openmetadata-ui/src/main/resources/ui/src/components/Database/Profiler/DataObservability/DataObservabilityTab.test.tsx",
       "testResults": [
         {
-          "ancestorTitles": [
-            "DataObservabilityTab",
-            "Rendering"
-          ],
+          "ancestorTitles": ["DataObservabilityTab", "Rendering"],
```

Comment:
@anuj-kumary can we remove this file altogether. I think this was added by mistake.